modules/virt-cluster-resource-requirements.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+[id="virt-cluster-resource-requirements_{context}"]
+= Additional hardware requirements for {VirtProductName}
+
+
+{VirtProductName} is an add-on to {product-title} and imposes additional overhead that you must account for when planning a cluster. Each cluster machine must accommodate the following overhead requirements in addition to the {product-title} requirements.
+
+[id="memory-overhead_{context}"]
+=== Memory overhead
+
+.Cluster memory overhead
+
+----
+Memory overhead per infrastructure node = 158 MiB
```

Comment:
Can we avoid the equals sign, and the least significant digit? These numbers are far from precises or constant. I'd feel more comfortable with `Memory overhead per infrastructure node ≈ 150 MiB`

Reply:
@dankenigsberg I have made changes per your suggestion.